README.md

```diff
@@ -1 +1,17 @@
 # TLA+ Foundation
+
+# 1. Goals of the Foundation
+The TLA+ Foundation (TLAF) is an independent, non-profit organization dedicated to fostering the adoption of the TLA+ specification language in industry, academia, and education, guided by the overall goal of advancing the practice of mathematical thinking in software engineering. The TLAF funds development related to TLA+ to broaden its use by practitioners and facilitate a thriving TLA+ community.  If not actively involved in work, the TLAF aims to create synergies among all community players.
```

Comment:
Most people now associate "mathematical thinking" with category theory, haskell, and typed FP. This could lead to either confusion or pressure to more in a more CT direction. I don't have a solution for this, just raising it as a possible issue.